routes/index.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function (app) {
```

Comment:
I guess you should stick to one convension either you use normal function or arrow function